starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <link rel="stylesheet"  href="css/main.css">
   <title>Apple Pie Recipe</title>
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
+
+<header>
+  <div id="AppleBackgroundImage">
```

Comment:
When we have to naming the class or id in HTML/CSS, we use the [KebabCase](http://wiki.c2.com/?KebabCase).
When we have to naming variables or functions on JS, we use [lowerCamelCase](https://en.wikipedia.org/wiki/Camel_case).